pkg/watermark/fetch/processor_manager.go

```diff
@@ -161,7 +161,7 @@ func (v *ProcessorManager) startHeatBeatWatcher() {
 			case store.KVPut:
 				// do we have such a processor
 				p := v.GetProcessor(value.Key())
-				if p == nil {
+				if p == nil || p.IsDeleted() {
```

Comment:
This is what I thought.

Reply:
if p.Deleted == true, how will a kvput event come through?